zon-format/tests/unit/test_delta.py

```
@@ -107,12 +108,41 @@ def test_deep_nesting(self):
+            {'v': 3579.16},     # benchmark regression case
+            {'v': math.pi},     # 17-sig-digit irrational
+            {'v': math.e},      # 17-sig-digit irrational
+            {'v': 0.1 + 0.2},   # classic non-terminating binary: 0.30000000000000004
+            {'v': -42.5},       # negative, crosses zero in deltas
+            {'v': 1e-10},       # small exponent
+            {'v': 1e15},        # large exponent
+        ]
+
+        decoded = self.decoder.decode(self.encoder.encode(data))
+
+        for original, got in zip(data, decoded):
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)